topics/appendix/tools/tools_gradle_intellij_plugin.md

```diff
@@ -0,0 +1,1210 @@
+
+> For more details, examples or issues reporting, go to the [IntelliJ Plugin Verifier](https://github.com/JetBrains/intellij-plugin-verifier) repository.
+>
+{type="tip"}
+
+> TIP To run Plugin Verifier in [`-offline`](https://github.com/JetBrains/intellij-plugin-verifier/pull/58) mode, set the Gradle [`offline` start parameter](https://docs.gradle.org/current/javadoc/org/gradle/StartParameter.html#setOffline-boolean-).
+>
+{type="tip"}
+
+
+### ideVersions
+IDEs to check, in `intellij.version` format, i.e.: `["IC-2019.3.5", "PS-2019.3.2"]`.
+Check the available build versions on [IntelliJ Platform Builds list](https://jb.gg/intellij-platform-builds-list).
+
+**Type:** `List<String>`
+
+**Default value:** output of the [`listProductsReleases`](#listproductsreleases-task) task
+
+
+### verifierVersion
+IntelliJ Plugin Verifier version, by default uses the latest available.
+It's recommended to use always the latest version.
+
+**Type:** `String`
+
+**Default value:** `latest`
+
+
+### verifierPath
+IntelliJ Plugin Verifier local path to the pre-downloaded jar file.
+If set, `verifierVersion` is ignored.
+
+**Type:** `String`
+
+**Default value:** path to the JAR file resolved using the `verifierVersion` property
+
+
+### localPaths
+A list of the paths to locally installed IDE distributions that should be used for verification in addition to those specified in `ideVersions`.
+
+**Type:** `List<File>`
+
+**Default value:** `[]`
+
+
+### distributionFile
+JAR or ZIP file of the plugin to verify.
+If empty, the task will be skipped.
+
+**Type:** `File`
+
+**Default value:** output of the `buildPlugin` task
+
+
+### failureLevel
+Defines the verification level at which task should fail if any reported issue will match.
+Can be set as `FailureLevel` enum or `EnumSet<FailureLevel>`.
+
+**Type:** `FailureLevel`
+
+**Default value:** `FailureLevel.COMPATIBILITY_PROBLEMS`
+
+
+### verificationReportsDir
+The path to directory where verification reports will be saved.
+
+**Type:** `String`
+
+**Default value:** `${project.buildDir}/reports/pluginVerifier`
+
+
+### downloadDir
+The path to directory where IDEs used for the verification will be downloaded.
+
+**Type:** `String`
+
+**Default value:** `System.getProperty("plugin.verifier.home.dir")/ides` or `System.getProperty("user.home")/.pluginVerifier/ides` or system temporary directory.
+
+
+### jbrVersion
+Custom JBR version to use for running the IDE.
+
+**Type:** `String`
+
+**Default value:** `null`
+
+**Accepted values:**
+- `8u112b752.4`
+- `8u202b1483.24`
+- `11_0_2b159`
+
+All JetBrains Java versions are available at JetBrains Space Packages, and [GitHub](https://github.com/JetBrains/JetBrainsRuntime/releases).
+
+
+### jbrVariant
```

Comment:
it seems the JBR explanations are double on this page now. can we extract them?? also deduplicate with existing https://plugins.jetbrains.com/docs/intellij/ide-development-instance.html#using-a-jetbrains-runtime-for-the-development-instance